### ISSUE TITLE
RSC Compatability

### ISSUE THREAD

**Issue by IanWONeill** [zeruth/OSRS-Deobfuscator#1]
Are you or anybody forking this able to create a branch for this to be able to deobfuscate the RSC client before Classic is shut down for good? The last revision is 235.

I've attached the latest gamepack and cache for testing if anybody decides to test this.
[rsc-235.zip](https://github.com/zeruth/OSRS-Deobfuscator/files/2043503/rsc-235.zip)

p.s. an error is thrown on writing the deobbed jar when I attempted this.

**Comment by zeruth** [comment]
https://github.com/zeruth/OSRS-Deobfuscator/commit/f084819cc1e44d9142022615c4a8ba72d273bdf3

That should take care of it. add rsc to end of args for rsc compatibility. ex: "Deob vanilla.jar deob.jar rsc"
Will leave this issue open until someone can verify the integrity of the output jar,

**Comment by IanWONeill** [comment]
Brilliant @zeruth I'm going to test it now.

**Comment by IanWONeill** [comment]
(y)

**Comment by zeruth** [comment]
Yeah I noticed that as well. And I believe it has to do with the same reason the deobfuscator wasn't working. I'm pretty sure RSC is compiled using an older version of java. FernFlower gets stuck, and other decompilers spit out a bunch of errors. But the client itself runs properly so i'm willing to bet its on the decompiler side of things.

**Comment by IanWONeill** [comment]
[client.zip](https://github.com/zeruth/OSRS-Deobfuscator/files/2046276/client.zip)

After reading what you said I ran it through JAD. looks a little better but still alot broken.
